controllers/buildpipeline/buildpipeline_controller_test.go

```diff
@@ -41,14 +41,14 @@ import (
 
 var _ = Describe("PipelineController", func() {
 	var (
-		manager            ctrl.Manager
-		pipelineReconciler *Reconciler
-		scheme             runtime.Scheme
-		req                ctrl.Request
-		successfulTaskRun  *tektonv1beta1.TaskRun
-		testpipelineRun    *tektonv1beta1.PipelineRun
-		hasApp             *applicationapiv1alpha1.Application
-		hasComp            *applicationapiv1alpha1.Component
+		manager                ctrl.Manager
+		pipelineReconciler     *Reconciler
+		scheme                 runtime.Scheme
+		req                    ctrl.Request
+		successfulTaskRun      *tektonv1beta1.TaskRun
+		integrationpipelineRun *tektonv1beta1.PipelineRun
```

Comment:
Would it make more sense to name this as `buildPipelineRun` since its type is `build`?

Reply:
Yes, thanks!